src/qml/main.qml

```diff
@@ -152,6 +152,14 @@ Kirigami.ApplicationWindow {
                     switchToPage(idleModePage);
                 }
             },
+            Kirigami.Action {
+                text: qsTr("Settings");
+                icon.name: ":/org/kde/kirigami/icons/settings-configure.svg";
+
+                onTriggered: {
+                    showMessageBox(qsTr("Sorry"), qsTr("This page is coming soon..."));
+                }
+            },
             Kirigami.Action {
```

Comment:
i'm thinking we probably want the settings to go on the other side of the separator (which, yeah, i can't help but think it'd be nice to have an explicit spacer, instead of just putting in an empty action, but we don't ;) ). Otherwise good! :)

Reply:
Will change in master as agreed elsewhere :)